src/main/java/seedu/brokeMan/entry/EntryDateComparator.java

```diff
@@ -0,0 +1,9 @@
+package seedu.brokeMan.entry;
+
+import java.util.Comparator;
+
+public class EntryDateComparator implements Comparator<Entry> {
+    public int compare(Entry e1, Entry e2) {
+        return e2.time.compareTo(e1.time);
+    }
+}
```

Comment:
Though not necessary, you could consider just including the two comparators within `EntryList`, since they are each only used once.